### PR TITLE
Update Kafo to 3.0.0

### PR DESCRIPTION
**Pull request by ekohl**
Not sure if this needs a bump of the foreman installer at the same time, but they probably need to be merged close together.

### CODE REVIEW

**Comment by ekohl** [comment]
This looks like an infra issue:

```
17:21:58 -- Creating Debian source package from ./kafo-3.0.0.tar.gz ...
17:22:10 /usr/lib/ruby/vendor_ruby/gem2deb/dh_make_ruby.rb:149:in `stat': No such file or directory @ rb_file_s_stat - /home/jenkins/.cache/gem2deb/gem_to_packages.yaml.new (Errno::ENOENT)
17:22:10 	from /usr/lib/ruby/vendor_ruby/gem2deb/dh_make_ruby.rb:149:in `generate_or_update_gem_to_package_data'
17:22:10 	from /usr/lib/ruby/vendor_ruby/gem2deb/dh_make_ruby.rb:66:in `initialize'
17:22:10 	from ../../gem2deb:96:in `new'
17:22:10 	from ../../gem2deb:96:in `<main>'
17:22:10 Build step 'Execute shell' marked build as failure
```

File: dependencies/bionic/kafo/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-kafo (3.0.0-1) stable; urgency=low
```
Comment (zjhuntin) [inline dependencies/bionic/kafo/changelog]: Was this generated from a script? I guess I'm not sure what the urgency part indicates here, or is it just to say "Hey this isn't needed RIGHT NOW"?

**Comment by mmoll** [comment]
[test packaging]

File: dependencies/bionic/kafo/changelog (hunk 1 above)
Comment (evgeni) [reply dependencies/bionic/kafo/changelog]: debian changelogs contain an `urgency` tag, to indicate how quickly packages should flow from `unstable` to `testing`. `low` means 10 days delay, `medium` 5 and `high` 2. as we don't upload to Debian proper, this field is irrelevant for us (but it needs to be set for the changelog entry to be valid)

Comment (ekohl) [reply dependencies/bionic/kafo/changelog]: And to answer the generated part: https://github.com/theforeman/foreman-packaging/blob/deb/develop/scripts/changelog.rb

Comment (zjhuntin) [reply dependencies/bionic/kafo/changelog]: Thanks for the knowledge!

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**